README.md

```diff
@@ -60,27 +60,41 @@ If `RMW_FASTRTPS_PUBLICATION_MODE` is not set, then both `rmw_fastrtps_cpp` and
 
 ### Full QoS configuration
 
-With `rmw_fastrtps`, it is possible to fully configure Fast DDS using an XML file as described in [Fast DDS documentation](https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/xml_configuration.html).
-When configuring the middleware using XML files, there are certain points that have to be taken into account:
+Fast DDS QoS policies can be fully configured through a combination of the [`rclcpp`/`rclpy` QoS] API, and the [Fast DDS XML] file's QoS elements. Configuraton depends on the environment variable `RMW_FASTRTPS_USE_QOS_FROM_XML`.
```

Comment:
@briansoe66 as I mentioned before, I'd rather not target specific client libraries. You can always link against the [`rmw`](https://github.com/ros2/rmw) interface layer for API documentation.